commcare_connect/opportunity/views.py

```diff
@@ -1130,11 +1146,11 @@ def user_visit_review(request, org_slug, opp_id):
         if review_status in [VisitReviewStatus.agree.value, VisitReviewStatus.disagree.value]:
             user_visits.update(review_status=review_status)
             update_payment_accrued(opportunity=opportunity, users=[visit.user for visit in user_visits])
-
+    RequestConfig(request, paginate={"per_page": 15}).configure(table)
```

Comment:
Wouldn't below style be more obvious?
```
table.paginate(page=request.GET.get("page", 1), per_page=15)
```

Reply:
It could even be simpler if you convert the view to class base view. Where you can simply set `paginate_by` on the view. That could also help clear the table and filter instantiation code.

Reply:
The code I included to paginate the table was suggested in the documentation for `django-tables` library.

Reply:
plus one to doing it the way recommended in the docs.